source/server/admin/admin.cc

```diff
@@ -760,11 +761,9 @@ Http::ServerConnectionPtr AdminImpl::createCodec(Network::Connection& connection
 
 bool AdminImpl::createNetworkFilterChain(Network::Connection& connection,
                                          const std::vector<Network::FilterFactoryCb>&) {
-  // Don't pass in the overload manager so that the admin interface is accessible even when
-  // the envoy is overloaded.
```

Comment:
Adjust comment so it explains why null_overload_manager_ is used despite the fact that we could pass in the real one.